apps/module-scan/src/store.ts

```diff
@@ -400,6 +401,17 @@ export default class Store {
     )
   }
 
+  async getBallotPaperSizeForElection(): Promise<Optional<BallotPaperSize>> {
+    const electionDefinition = await this.getElectionDefinition()
+    if (!electionDefinition) {
+      return
+    }
+    if (!electionDefinition.election.ballotLayout?.paperSize) {
+      return BallotPaperSize.Letter
+    }
+    return electionDefinition.election.ballotLayout.paperSize
+  }
```

Comment:
If you don't need this to be `undefined` sometimes and use a default other times, you can do this:

```suggestion
  async getBallotPaperSizeForElection(): Promise<BallotPaperSize> {
    const electionDefinition = await this.getElectionDefinition()
    return (
      electionDefinition?.election.ballotLayout?.paperSize ??
      BallotPaperSize.Letter
    )
  }
```

Reply:
Yeah in practice this is never called unless electionDefinition is defined so I'm not sure how much it matters how that case is handled.